.devcontainer/typescript/devcontainer.json

```
@@ -0,0 +1,24 @@
+{
+	"name": "TypeScript",
+	"dockerComposeFile": [
+	  "./docker-compose.yml"
+	],
+	"workspaceFolder": "../typescript",
+	"service": "worker",
+	"remoteUser": "vscode",
```

vscode user does not exist in Docker image `node:20`. Use node insted of vscode.
 
```suggestion
	"remoteUser": "node",
```